### PR TITLE
Zalinasbranch

### PR DESCRIPTION
**Pull request by ZalinaKabulova**
image

### CODE REVIEW

**Review by muazyagci — APPROVED** [review]
Perfect

File: src/test/java/NewClass.java [hunk 1]
```diff
@@ -2,5 +2,8 @@ public class NewClass {
     public static void main(String[] args) {
 
         System.out.println("bhbj");
+        System.out.println("nkfnkd");
+        System.out.println("nkfnkd");
+
```
Comment (KaterynaHania) [inline src/test/java/NewClass.java:7]: unnecessary line, need to be deleted

Comment (Rahman16-dev) [reply src/test/java/NewClass.java:7]: Ok :( I'll deal with it

File: src/test/java/NewClass.java [hunk 2]
```diff
@@ -2,5 +2,8 @@ public class NewClass {
     public static void main(String[] args) {
 
         System.out.println("bhbj");
+        System.out.println("nkfnkd");
+        System.out.println("nkfnkd");
```
Comment (esperast) [inline src/test/java/NewClass.java:6]: Need to be edited!

File: src/test/java/NewClass.java (hunk 1 above)
Comment (Betululutin) [reply src/test/java/NewClass.java:7]: Perfect! Well done..

**Review by esperast — COMMENTED** [review]
Needs to be edited!

**Pull request merged**